setup.py

```diff
@@ -2003,11 +2003,11 @@ def detect_ctypes(self, inc_dirs, lib_dirs):
             ffi_inc = find_file('ffi.h', [], inc_dirs)
         if ffi_inc is not None:
             ffi_h = ffi_inc[0] + '/ffi.h'
+            # Just check that the file is not empty
```

Comment:
Looks like the real origin is [bpo-1464444](https://bugs.python.org/issue1464444), in which @doko42 noted that the check for `#define LIBFFI_H` may or may not be worth much.  At this point, I don't think we need to even check that there is anything in the file, just that `ffi.h` exists.